src/analysis/retail/hunter/marksmanship/CHANGELOG.tsx

```diff
@@ -5,6 +5,11 @@ import TALENTS from 'common/TALENTS/hunter';
 import ITEMS from 'common/ITEMS';
 import SPELLS from 'common/SPELLS';
 export default [
+  change(date(2024, 11, 15), <>Added reset logic for <SpellLink spell={SPELLS.RAPID_FIRE} /> and <SpellLink spell={TALENTS.AIMED_SHOT_TALENT} /> from <SpellLink spell={SPELLS.WAILING_ARROW_DAMAGE} /> and <SpellLink spell={TALENTS.READINESS_TALENT} /> </>, Yellot),
+  change(date(2024, 11, 15), <>Added <SpellLink spell={SPELLS.RAPID_FIRE}/> reset logic from <SpellLink spell={TALENTS.SURGING_SHOTS_TALENT} /></>, Yellot),
+  change(date(2024, 11, 15), <>Simplified reset logic inside of <SpellLink spell={TALENTS.LOCK_AND_LOAD_TALENT} /></>, Yellot),
+  change(date(2024, 11, 15), <>Inclusion of <SpellLink spell={TALENTS.TACTICAL_RELOAD_TALENT} /> and <SpellLink spell={SPELLS.STEADY_FOCUS_BUFF} /> for CDR logic in Aimed Shot</>, Yellot),
+  change(date(2024, 11, 15), <>Add <SpellLink spell={TALENTS.TACTICAL_RELOAD_TALENT}/> constant for CDR Reduction</>, Yellot),
```

Comment:
When making a lot of changes to the same category of thing, you only need to add one changelog entry

Reply:
gotcha, consolidated all of em.